VATSpy.dat

```diff
@@ -18357,8 +18363,8 @@ BICC|Iceland Radio|BIRD-E,BIRD-N,BIRD-S,BIRD-W
 CARI|Caribbean Control|MYNN,MUFH,MKJK,MTEG,MDCS,TNCF,TJZS,TTZP,KZMO
 CARE|Caribbean Control East|MDCS,TNCF,TJZS,TTZP
 CARW|Caribbean Control West|MYNN,MUFH,MKJK,MTEG,KZMO
-EDUU|Rhein Radar|EDGG-E,EDMM-G,EDMM-R
-EDYY|Maastricht Radar|EBBU,EHAA,EDGG-P,EDWW-A,EDWW-B
+EDUU|Rhein Radar|EDGG-E,EDMM-G,EDMM-R,EDMM-Z,EDWW-B

```

Comment:
You renamed EDMM-R to EDMM-Z above, please do it here too.

Reply:
In the FIRBoundaries.dat are both Z and R

Reply:
Sorry my bad, github just had a poor way of displaying the change at line 2997.